snippets/csharp/VS_Snippets_Misc/wifcustomtokenst/cs/simplewebtoken/simplewebtoken.cs

```diff
@@ -28,18 +28,18 @@ public class SimpleWebToken : SecurityToken
 
         NameValueCollection _properties;
         //</Snippet3>
-        string _serilaizedToken;
+        string _serializedToken;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="SimpleWebToken"/> class.
-        /// This is internal contructor is only called from the <see cref="SimpleWebTokenHandler"/> when reading a token received from the wire.
+        /// This is internal constructor is only called from the <see cref="SimpleWebTokenHandler"/> when reading a token received from the wire.
```

Comment:
Just noticed this if you'd like to commit my suggestion while you're at it
```suggestion
         /// This is an internal constructor that is only called from the <see cref="SimpleWebTokenHandler"/> when reading a token received from the wire.
```